src/ArrayUtil.php

```diff
@@ -21,6 +21,12 @@ public static function flattenMap(callable $callable, array $array): array
      */
     public static function flatten(array $array): array
     {
+        if ($array === null) {
```

Comment:
`$array` will never be `null` because of parameter type hint on line 22.

Reply:
Agreed.